db/migrate/20210122185518_add_default_aal_to_service_providers.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultAalToServiceProviders < ActiveRecord::Migration[6.0]
+  def change
+    add_column :service_providers, :default_aal, :integer
```

Comment:
do we need to add the same aal -> default_aal copy as we have in the IDP PR?

Reply:
Nope - there isn't an existing `aal` field on the dashboard 🙂

Reply:
Are we always expecting this to be needed, e.g. should we have a `null: false` and a default set?

Reply:
`null` is fine - currently, only two SPs have anything set at all in the existing `aal` field.